components/CommonHeader.tsx

```diff
@@ -17,11 +17,11 @@ import { theme } from 'theme/theme'
 import OSLogo from '../assets/darkThemeLogo.svg'
 
 const page_mapping = {
-  About: '/about_new',
-  Projects: '/projects_new',
-  Partners: '/partners_new',
+  About: '/about',
```

Comment:
this has to be the about_new as well for now...

Reply:
It shouldn't be a problem to merge

Reply:
It looks like vercel is throwing a build error about identical path names:

> error - Conflicting paths returned from getStaticPaths, paths must be unique per page.
> --
> 12:57:02.828 | See more info here: https://nextjs.org/docs/messages/conflicting-ssg-paths
> 12:57:02.829 |  
> 12:57:02.829 | path: "/about" from page: "/[slug]" conflicts with path: "/about"

Reply:
Looks like some configuration in Sanity is causing this.  How do I make changes there.

Reply:
per my last comment. Because we don't need to preserve any of the old pages on this branch ... since this branch would only get merged to main as we publish the website completely, we can just remove those pages. They don't need to exist in your branch.

Reply:
I removed the "*_new" pages and it still fails.  Any ideas?